### PR TITLE
MINOR: Updated cassandra tutorial index.md

### PR DESCRIPTION
**Pull request by prashantgoel**
## Description
Its a very nice documentation, but while connecting to the cluster using the cqlsh client by hitting below command:
`docker run cassandra:2.2.5 cqlsh <HOST>`

I got below error:
Connection error: ('Unable to connect to any servers', {'10.0.10.16': ProtocolError("cql_version '3.3.1' is not supported by remote (w/ native protocol). Supported versions: [u'3.4.0']",)})

To resolve this issue, we need to force cqlsh to use a specific cql version using the flag `--cqlversion=3.4.0`, so we need to hit below command:
`docker run -ti cassandra:2.2.5 cqlsh --cqlversion=3.4.0 <HOST>`

or we run connect to cluster using the cqlsh client by using the latest image:
`docker run -ti cassandra:latest cqlsh <HOST>`

## Urgency
- [ ] Blocker 
- [ ] High
- [x] Medium

## Requirements
- Test all commands and procedures.
- Build content [locally](https://github.com/dcos/dcos-docs#test-local) and test for formatting/links.
- Add redirects to [dcos-website/redirect-files](https://github.com/dcos/dcos-website#managing-redirects).
- Change all affected versions (e.g. 1.7, 1.8, and 1.9).
- See the [contribution guidelines](https://github.com/dcos/dcos-docs#contributing).

### CODE REVIEW

**Comment by prashantgoel** [comment]
Please review !!

**Comment by emanic** [comment]
@prashantgoel Very sorry for the delay, it seems that this one slipped through the cracks. @sascala @gabrielhartmann can you please verify this update? Thanks!

File: 1.7/usage/tutorials/cassandra/index.md [hunk 1]
```diff
@@ -146,13 +146,13 @@ core@ip-10-0-6-153 ~ $
 You are now inside your DC/OS cluster and can connect to the Cassandra cluster directly. Connect to the cluster using the cqlsh client:
 
 ```bash
-core@ip-10-0-6-153 ~ $ docker run cassandra:2.2.5 cqlsh <HOST>
+core@ip-10-0-6-153 ~ $ docker run -ti cassandra:latest cqlsh <HOST>
```
Comment (gabrielhartmann) [inline 1.7/usage/tutorials/cassandra/index.md]: ususally go with `-it`, as it sounds better when you read it out loud.  Not sure we maintain a `:latest` docker image...  Did you confirm what version that actually is?

Comment (sascala) [inline 1.7/usage/tutorials/cassandra/index.md]: I'm testing as soon as my cluster comes up. Thanks for taking a look.

Comment (prashantgoel) [inline 1.7/usage/tutorials/cassandra/index.md]: Latest verion: 3.10.
I have made this change i.e. "-ti", as same has been used at other places in the documentation.

**Comment by prashantgoel** [comment]
@gabrielhartmann @sascala : I have made the changes as per your suggestions. Please check !!

**Comment by prashantgoel** [comment]
@emanic : Please review !!

**Comment by sascala** [comment]
Thank you, @prashantgoel

**Pull request merged**